src/coreclr/jit/emitxarch.cpp

```diff
@@ -5910,7 +5935,12 @@ void emitter::emitIns_R(instruction ins, emitAttr attr, regNumber reg)
     noway_assert(emitVerifyEncodable(ins, size, reg));
 
     UNATIVE_OFFSET sz;
-    instrDesc*     id = emitNewInstrSmall(attr);
+    instrDesc*     id  = emitNewInstrSmall(attr);
```

Comment:
this change can be reverted I suppose?

Reply:
I did not add this. For soem reason, jit format fails in CI and the patch created has this